src/assets/styles/css/complex-portal-theme.scss

```diff
@@ -295,3 +276,30 @@ a:visited {
 }
 
 /*********** END HOME PAGE SECTION *******************************/
+
+
+.predicted {
```

Comment:
I grouped every predicted special case in a single parent using scss, it was more handy than adding `.predicted` before every item ^^